commit 8198ffe192d1450962f59282c68b695964f3f6f2
Author: Kay Lin <kaymw@aosc.io>
Date:   Wed Nov 14 20:55:18 2018 -0800

    Use sudo for fgconsole
    
    AOSC OS's fgconsole requires root permission to work correctly.
    
    Signed-off-by: Kay Lin <kaymw@aosc.io>

diff --git a/nvidia-xrun b/nvidia-xrun
index 40eaecd..479a517 100644
--- a/nvidia-xrun
+++ b/nvidia-xrun
@@ -30,7 +30,7 @@ if [ "$1" == "-d" ]
 fi
 
 # calculate current VT
-LVT=`fgconsole`
+LVT=`sudo fgconsole`
 
 # calculate first usable display
 XNUM="-1"
